### PR TITLE
Update my recipe2

### PR DESCRIPTION
**Pull request by sherkinglee**
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by sherkinglee** [comment]
@BiocondaBot please add label

**Comment by sherkinglee** [comment]
Hi, I am a newer to bioconda and I wonder what should I do next? How do I add more commits by pushing to the update_my_recipe2 branch on sherkinglee/bioconda-recipes?  Thanks for your help!

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@sherkinglee Yes, you can push them to the `update_my_recipe2` branch on your fork if you want (I'll hold off on merging this then). Make sure to `git pull` that first, since you can see I made a change to it. Otherwise if you're finished making changes let me know so I can merge these in.

**Comment by sherkinglee** [comment]
Hi, there is nothing need to be done actually. Thanks for your help and please merge this branch.

File: recipes/ribominer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    - cAIPlot=RiboMiner.cAIPlot:main
+    - ModifyHTseq=RiboMiner.ModifyHTseq:main
+    - ReadsLengthOfSpecificRegions=RiboMiner.ReadsLengthOfSpecificRegions:main
+    - CoverageOfEachTrans=RiboMiner.CoverageOfEachTrans:main
+    - PlotTransCoverage=RiboMiner.PlotTransCoverage:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - biopython >=1.70
```
Comment (bgruening) [inline recipes/ribominer/meta.yaml:60]: Can you remove everything from the host section besides python and pip.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**